crates/red_knot_python_semantic/resources/mdtest/narrow/conditionals_is.md

```diff
@@ -37,4 +41,20 @@ reveal_type(y)  # revealed: bool
 if y is x is False:  # Interpreted as `(y is x) and (x is False)`
     reveal_type(x)  # revealed: Literal[False]
     reveal_type(y)  # revealed: bool
+else:
+    reveal_type(x)  # revealed: bool
```

Comment:
No, because there are two conditions in the `if` clause. And only one of them could be `False`: if `y = True` and `x = False`, we also end up in this branch.

Reply:
Unresolving this conversation because it might be useful to add a comment here explaining this?